docs/install/create-project-classic-ui.md

```diff
@@ -0,0 +1,224 @@
+```
+
+```console
+% pipx run cookieplone backend_addon
+╭─────────────────────────────────── cookieplone ────────────────────────────────────╮
+│                                                                                    │
+│                                  .xxxxxxxxxxxxxx.                                  │
+│                              ;xxxxxxxxxxxxxxxxxxxxxx;                              │
+│                           ;xxxxxxxxxxxxxxxxxxxxxxxxxxxx;                           │
+│                         xxxxxxxxxx              xxxxxxxxxx                         │
+│                       xxxxxxxx.                    .xxxxxxxx                       │
+│                      xxxxxxx      xxxxxxx:            xxxxxxx                      │
+│                    :xxxxxx       xxxxxxxxxx             xxxxxx:                    │
+│                   :xxxxx+       xxxxxxxxxxx              +xxxxx:                   │
+│                  .xxxxx.        :xxxxxxxxxx               .xxxxx.                  │
+│                  xxxxx+          ;xxxxxxxx                 +xxxxx                  │
+│                  xxxxx              +xx.                    xxxxx.                 │
+│                 xxxxx:                      .xxxxxxxx       :xxxxx                 │
+│                 xxxxx                      .xxxxxxxxxx       xxxxx                 │
+│                 xxxxx                      xxxxxxxxxxx       xxxxx                 │
+│                 xxxxx                      .xxxxxxxxxx       xxxxx                 │
+│                 xxxxx:                      .xxxxxxxx       :xxxxx                 │
+│                 .xxxxx              ;xx.       ...          xxxxx.                 │
+│                  xxxxx+          :xxxxxxxx                 +xxxxx                  │
+│                  .xxxxx.        :xxxxxxxxxx               .xxxxx.                  │
+│                   :xxxxx+       xxxxxxxxxxx              ;xxxxx:                   │
+│                    :xxxxxx       xxxxxxxxxx             xxxxxx:                    │
+│                      xxxxxxx      xxxxxxx;            xxxxxxx                      │
+│                       xxxxxxxx.                    .xxxxxxxx                       │
+│                         xxxxxxxxxx              xxxxxxxxxx                         │
+│                           ;xxxxxxxxxxxxxxxxxxxxxxxxxxxx+                           │
+│                              ;xxxxxxxxxxxxxxxxxxxxxx;                              │
+│                                  .xxxxxxxxxxxxxx.                                  │
+│                                                                                    │
+╰────────────────────────────────────────────────────────────────────────────────────╯
+╭─────────────────────────────────── Plone Addon ────────────────────────────────────╮
+│ Creating a new Plone Addon                                                         │
```

Comment:
This is outside the scope of this PR, but: The backend_addon template is equally useful for creating an addon or a Classic UI project. We should update its name and the wording of its defaults so it's less addon-centric, which might be confusing in the context of this chapter.

Reply:
Created issue: https://github.com/plone/cookieplone-templates/issues/61